make/test_target_script.sh

```diff
@@ -27,15 +27,27 @@ LIGHT_CYAN='\033[1;36m'
 BOLD='\033[1m'
 NC='\033[0m'
 
+QUIET="${QUIET:-0}"
+if [ "$QUIET" -eq 1 ] && [ "${WSLcross}" != "true" ]; then
+    # cth_log_redirect is enabled by default, to configure it we need to remove and re-add it
+    ARGS="-enable_builtin_hooks false -ct_hooks cth_log_redirect [{mode,replace}] ${ARGS}"
```

Comment:
this `-ct_hooks` overrides(or shadows if used twice) other use of `-ct_hooks` in command line.
So this command with your PR:
`make TEST_NEEDS_RELEASE=false ssl_test ARGS="-suite ssl_api_SUITE -ct_hooks obs_cth"`
will not install obs_cth.

Can you maybe check if we could append hook modules?

Reply:
I believe de3808b9a7d7f225e0d059ba6c3a82fd390dca29 _should_ solve this, although `ct_run`s option parsing is still a bit beyond me..